engines/sci/detection.cpp

```diff
@@ -85,11 +85,11 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"lsl3",            "Leisure Suit Larry 3: Passionate Patti in Pursuit of the Pulsating Pectorals"},
 	{"mothergoose",     "Mixed-Up Mother Goose"},
 	{"pq2",             "Police Quest II: The Vengeance"},
-	{"qfg1",            "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down)
+	{"qfg1",            "Quest for Glory: So You Want to Be a Hero"},	// Note: There was also a SCI11 VGA remake of this (further down)
 	{"sq3",             "Space Quest III: The Pirates of Pestulon"},
 	// === SCI01 games ========================================================
 	{"qfg2",            "Quest for Glory II: Trial by Fire"},
-	{"kq1sci",          "King's Quest I: Quest for the Crown"},	// Note: There was also an AGI version of this
+	{"kq1sci",          "King's Quest: Quest for the Crown"},	// Note: There was also an AGI version of this
```

Comment:
The remake was titled "Roberta Williams' King's Quest I: Quest for the Crown"
https://www.mobygames.com/game/dos/roberta-williams-kings-quest-i-quest-for-the-crown/cover-art/gameCoverId,725055/

Reply:
Thank you for fact checking me on all of these; you are definitely right thank you.